app/controllers/activities_controller.rb

```diff
@@ -49,6 +49,7 @@ def show
           tab_name: current_tab
         )
         @commitment = CommitmentPresenter.new(@activity.commitment)
+        @tab_name = current_tab
```

Comment:
I wonder in fact if I shouldn't just make a `helper_method :current_tab`, drop this line and use `current_tab == tab` in https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/12bba9f0ea1c4748ccc3b3f77500a45baf9d5519/app/views/shared/activities/_tab_nav_item.html.haml#L1

Reply:
Yep, that does indeed feel substantially better